### ISSUE TITLE
Interferes with QEMU/KVM hidden control bars

### ISSUE THREAD

**Issue by RedBearAK** [marcinjahn/gnome-peek-top-bar-on-fullscreen-extension#15]
@marcinjahn 

Recently I had begun encountering a problem while using QEMU/KVM virtual machines in full screen mode, in Virt-Manager and GNOME Boxes. After taking some time to troubleshoot by disabling GNOME extensions one-by-one, the source of the issue appears to be this extension. 

Prior to GNOME 45 I was using Hide Top Bar and set it to have similar behavior to this extension, allowing access to the hidden top bar while using a full screen app, and it never interfered with the ability to reveal the control bars in Virt-Manager and GNOME Boxes. So this extension must be doing something different. I had installed this because Hide Top Bar still isn't officially available for G45. 

The control bars I'm talking about are basically the title bar with GNOME Boxes (which hides when you go full screen), and Virt-Manager provides a little pop-out in the center of the top edge of the screen. These provide a way to access the button that exits full screen mode with the mouse. With the Hide Top Bar extension it was always possible to get the control bars for the virtual machine to reveal before the mouse pressure would reveal the top bar. 

So basically I have to disable this extension in order to be able to get my VMs out of full screen mode or access the other tools on the VM control bars. 

I have no knowledge of how to do anything differently in your extension to keep this from being a problem. Perhaps the implementation of the pressure option would help.
